spec/system/managing_council_information_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "managing council information", type: :system do
+  let(:local_authority) { create(:local_authority, :default) }
+  let(:user) { create(:user, :administrator, local_authority: local_authority) }
+
+  it "allows the administrator to manage the reviewer group email" do
+    sign_in(user)
+    visit(administrator_dashboard_path)
+    row = find_all("tr").find { |tr| tr.has_content?("Manager group email") }
```

Comment:
Not in scope for this PR but I'm thinking we could move these specific finders to do with tables or any other relevant bits into an Rspec::Matcher class https://rubydoc.info/github/rspec/rspec-expectations/RSpec/Matchers to improve readability and for reuse

For another project, we did something like:

```ruby
RSpec::Matchers.define :have_table_row do |cells, row_index|
  match do |actual|
    page.within(:tr, row_index) do
      children = page.find_all(:xpath, "./*")
      expect(children.size).to eq(cells.size)

      cells.each_with_index do |cell, index|
        expect(page).to have_table_cell(cell, index + 1)
      end
    end
  end
end
```

Then in the spec you would just do ` expect(page).to have_table_row ....`

Reply:
That's an excellent idea! I will do.